hudi-common/src/main/java/org/apache/hudi/common/fs/FSUtils.java

```diff
@@ -79,7 +80,7 @@ public class FSUtils {
   private static final Logger LOG = LogManager.getLogger(FSUtils.class);
   // Log files are of this pattern - .b5068208-e1a4-11e6-bf01-fe55135034f3_20170101134598.log.1
   private static final Pattern LOG_FILE_PATTERN =
-      Pattern.compile("\\.(.*)_(.*)\\.(.*)\\.([0-9]*)(_(([0-9]*)-([0-9]*)-([0-9]*)(-cdc)?))?");
+      Pattern.compile("\\.(.*)_(.*)\\.(.*)\\.([0-9]*)(_(([0-9]*)-([0-9]*)-([0-9]*)))?");
```

Comment:
How about we add a `(.cdc)` group instead of `(-cdc)` here ?

Reply:
Only replacing `(-cdc)` to `(.cdc)` doesn't work, it has to change this `LOG_FILE_PATTERN` more.
This way will be easier to understand and make the affects clear.

Reply:
Changed to this pattern should work well:
```java
Pattern.compile("\\.(.*)_(.*)\\.(.*)\\.([0-9]*)(_(([0-9]*)-([0-9]*)-([0-9]*)(\\.cdc)?))?");
```

I have tested it locally and it works correctly.

Reply:
the pattern you provided doesn't work for filename with `.cdc`, like `.7a7aa1f6-370b-4871-a077-0848f1472e87-0_20221024114225320.log.1_0-125-118.cdc`.